config/clients/java/config.overrides.json

```diff
@@ -0,0 +1,34 @@
+{
+  "sdkId": "java",
+  "gitRepoId": "java-sdk",
+  "packageName": "dev.openfga:openfga-sdk",
+  "artifactId": "openfga-sdk",
+  "groupId": "dev.openfga",
+  "apiPackage": "dev.openfga.api",
+  "modelPackage": "dev.openfga.api.model",
+  "packageVersion": "0.0.1",
+  "snapshotVersion": false,
+  "packageDescription": "Java SDK for OpenFGA",
+  "artifactDescription": "Java SDK for OpenFGA",
+  "packageDetailedDescription": "This is an autogenerated java SDK for OpenFGA. It provides a wrapper around the [OpenFGA API definition](https://openfga.dev/api).",
+  "developerEmail": "community@openfga.dev",
+  "developerName": "OpenFGA Contributors",
+  "developerOrganization": "OpenFGA",
+  "developerOrganizationUrl": "https://openfga.dev",
+  "licenseName": "Apache-2.0",
+  "licenseUrl": "https://github.com/openfga/dotnet-sdk/blob/main/LICENSE",
```

Comment:
will not be dotnet-sdk

Reply:
Fixing

Reply:
Updated in [b9f329a](https://github.com/openfga/sdk-generator/pull/158/commits/b9f329aad7eeb2fce786213b7b3563eab0e43a57)